mention-bot.js

```diff
@@ -415,8 +420,8 @@ async function guessOwnersForPullRequest(
 ): Promise<Array<string>> {
   var diff = await fetch(repoURL + '/pull/' + id + '.diff');
   var files = parseDiff(diff);
-  var defaultOwners = getDefaultOwners(files, config.alwaysNotifyForPaths);
-
+  var defaultOwners = getMatchingOwners(files, config.alwaysNotifyForPaths);
+  var fallbackOwners = getMatchingOwners(files, config.fallbackNotifyForPaths);
```

Comment:
In case we don't wanna integrate this in the `findPotentialReviewers` (disabled) flow, should we have this being defined below? Otherwise we're running/setting it here for no reason.